src/strands/telemetry/metrics.py

```diff
@@ -151,6 +151,21 @@ def add_call(
             metrics_client.tool_error_count.add(1, attributes=attributes)
 
 
+@dataclass
+class EventLoopUsageMetric:
+    """Aggregated metrics for a single event loop cycle.
+
+    Attributes:
+        cycle_id: Current eventLoop cycle id.
+        usage: Total token usage for the entire cycle (sum of all model invocations).
+        model_usages: List of usage metrics for each model invocation, in chronological order.
```

Comment:
We should indicate when `usage != model_usages[0]`; all of the examples currently show them as the same and this doesn't indicate when it's not, given that a single cycle is "Model + Tool invocations"